docs: update rds db proxy access instructions

stevenGravy opened the pull request:
Error in configure command

stevenGravy committed:
docs: update rds db access instructions

greedy52 commented:
@stevenGravy the AWS region should be "optional" for RDS databases as they can be parsed from the database URI. Have you run into any issues without specifying the region?

stevenGravy committed:
docs: fix command for configure

stevenGravy commented:
> @stevenGravy the AWS region should be "optional" for RDS databases as they can be parsed from the database URI. Have you run into any issues without specifying the region?

I was using a sample uri, not a RDS one, so it didn't recognize for IAM generation. thanks @greedy52 . 

I reverted that and just fixed the `|` that was meant to be a `\`.
